README.md

```diff
@@ -62,11 +62,18 @@ As of `v0.1.0`, the component auto scrolls to the focused `TextInput` 😎. For
 
 In order to scroll to any `TextInput` field, you can use the built-in method `scrollToFocusedInput`. Example:
 
+```js
+import { findNodeHandle } from 'react-native';
+```
+
 ```js
 _scrollToInput (reactNode: any) {
   // Add a 'scroll' ref to your ScrollView
   this.scroll.props.scrollToFocusedInput(reactNode)
```

Comment:
This is outdated too now,
As mentioned here:
https://github.com/APSL/react-native-keyboard-aware-scroll-view/issues/553